go-controller/pkg/ovn/libovsdbops/acl.go

```diff
@@ -147,30 +147,32 @@ func CreateOrUpdateACLs(nbClient libovsdbclient.Client, acls ...*nbdb.ACL) error
 	return err
 }
 
-func UpdateACLLoggingOps(nbClient libovsdbclient.Client, ops []libovsdb.Operation, acl *nbdb.ACL) ([]libovsdb.Operation, error) {
+func UpdateACLsLoggingOps(nbClient libovsdbclient.Client, ops []libovsdb.Operation, acls ...*nbdb.ACL) ([]libovsdb.Operation, error) {
 	if ops == nil {
 		ops = []libovsdb.Operation{}
 	}
 
-	err := findACL(nbClient, acl)
-	if err != nil {
-		return nil, err
-	}
+	for _, acl := range acls {
+		err := findACL(nbClient, acl)
+		if err != nil {
+			return nil, err
+		}
 
-	uuid := acl.UUID
-	acl.UUID = ""
-	op, err := nbClient.Where(&nbdb.ACL{UUID: uuid}).Update(acl, &acl.Severity, &acl.Log)
-	if err != nil {
-		return nil, err
+		uuid := acl.UUID
+		acl.UUID = ""
```

Comment:
@maiqueb could you add a comment to explain why clearing the ACL UUID and then resetting it is needed?

Reply:
I have no clue why this is done; I just adapted the existing code and adapted it to receive variadic input. 

@jcaamano could you elaborate here why this is needed ? I don't mind adding it to the code.

Reply:
If the UUID was not cleared it would fail on update server side because it is immutable. Only for `Update`, for `Mutate` it is not needed so I guess this was copy/paste kind of thing. Anyway, this is no longer needed for `Update` as well since the recently merged https://github.com/ovn-org/libovsdb/pull/210 so if we pull this by vendoring it can be cleaned up for `Update`as well.